xsuite/src/world/fsworld.test.ts

```diff
@@ -944,6 +962,17 @@ test.concurrent("FSContract.getAccountBalance", async () => {
   expect(await contract.getAccountBalance()).toEqual(10n ** 18n);
 });
 
+test.concurrent("FSContract.getAccountValue", async () => {
+  using world = await FSWorld.start();
+  const { wallet } = await createAccounts(world);
+  const contract = await wallet.createContract({
```

Comment:
`world` could directly be used to create the contract. No need for `wallet`. Also `contract` just needs to have kvs, no code and codeMetadata needed (I don't think they are required?).